src/mkv.c

```diff
@@ -556,14 +556,14 @@ void get_markov_options(struct db_main *db,
 }
 
 void get_markov_start_end(char *start_token, char *end_token,
-                          unsigned long long mkv_max,
-                          unsigned long long *mkv_start, unsigned long long *mkv_end)
+                          uint64_t mkv_max,
+                          uint64_t *mkv_start, uint64_t *mkv_end)
 {
 	*mkv_start = 0;
 	*mkv_end = 0;
 
-	if ((start_token != NULL) && (sscanf(start_token, LLd, mkv_start) == 1)) {
-		if ((end_token != NULL) && (sscanf(end_token, LLd, mkv_end) == 1)) {
+	if ((start_token != NULL) && (sscanf(start_token, "%"PRId64, mkv_start) == 1)) {
+		if ((end_token != NULL) && (sscanf(end_token, "%"PRId64, mkv_end) == 1)) {
```

Comment:
You declare these as uint64_t so should use PRIu64 (yeah it was wrong before your change too)